### DISCUSSION TITLE
Storing aggregated metrics from previous rounds

### DISCUSSION THREAD

**Discussion by the-nishant** [FETS-AI/Challenge#59] [Q&A]
In an aggregation function, if we compute the square of the average gradient on the aggregator and want to use it in next round, how do we add it to the db_iterator?

**Comment by sarthakpati** [comment]
Hi @the-nishant,

Thank you for interest in the FeTS Challenge! For this answer, I am pinging @brandon-edwards @psfoley @alexey-gruzdev

**Reply by psfoley** [reply]
Hi @the-nishant, currently there isn't a great way to add custom information to the db_iterator from within the aggregation function. This is something we would like to support though. I can't promise we can implement a workaround for the current competition, but the FeTS organizing team will talk internally about it and get back to you
